### PR TITLE
Migrate to null-safety

### LINKED ISSUE #124: Upgrade dependencies of rx_bloc_generator
As a developer I want rx_bloc_generator to use the latest stable dependencies

- [x] build_runner
- [x] more ...

### PR DESCRIPTION
**Pull request by StanevPrime**
Closes #124

### CODE REVIEW

File: packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml [hunk 1]
```diff
@@ -23,13 +23,13 @@ dependencies:
...
   auto_route_generator: ^1.0.2
-  build_runner: ^1.12.2
+  build_runner:
   # flutter_driver:
   #   sdk: flutter
   flutter_test:
     sdk: flutter
   mockito: ^5.0.3
-  rx_bloc_generator: ^3.0.2
+  rx_bloc_generator:
```
Comment (PrimeSwiftArt) [inline packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml:32]: Why we don't have a version here?

File: packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml [hunk 2]
```diff
@@ -23,13 +23,13 @@ dependencies:
 
 dev_dependencies:
   auto_route_generator: ^1.0.2
-  build_runner: ^1.12.2
+  build_runner:
```
Comment (PrimeSwiftArt) [inline packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml:26]: Why we don't have a version here?

File: packages/rx_bloc/README.md [hunk 3]
```diff
@@ -86,7 +86,7 @@ Here is where the [rx_bloc_generator](https://pub.dev/packages/rx_bloc_generator
 ```dart
 dev_dependencies:
   build_runner:
-  rx_bloc_generator:^3.0.0
+  rx_bloc_generator:
```
Comment (PrimeSwiftArt) [inline packages/rx_bloc/README.md:89]: Why we don't have a version here?

File: packages/rx_bloc_generator/test/generator_test.dart [hunk 4]
```diff
@@ -1,4 +1,4 @@
-//@dart=2.9
+
```
Comment (PrimeSwiftArt) [inline packages/rx_bloc_generator/test/generator_test.dart:1]: Remove this empty line

File: packages/rx_bloc_list/pubspec.yaml [hunk 5]
```diff
@@ -18,7 +18,7 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  rx_bloc_generator: ^3.0.1
+  rx_bloc_generator:
```
Comment (PrimeSwiftArt) [inline packages/rx_bloc_list/pubspec.yaml:21]: Why we don't have a version here?

**Review by PrimeSwiftArt — CHANGES_REQUESTED** [review]

File: packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml (hunk 1 above)
Comment (StanevPrime) [reply packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml:32]: The motivation of not having a version of rx_bloc_generator is that this is a template and supposably should be using the latest compatible version

Comment (PrimeSwiftArt) [reply packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml:32]: Then why `mockito` has a version (`^3.0.2`)?

File: packages/rx_bloc_list/pubspec.yaml (hunk 5 above)
Comment (StanevPrime) [reply packages/rx_bloc_list/pubspec.yaml:21]: Now we have a version

File: packages/rx_bloc/README.md (hunk 3 above)
Comment (StanevPrime) [reply packages/rx_bloc/README.md]: Done

File: packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml (hunk 2 above)
Comment (StanevPrime) [reply packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml]: Done

File: packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml (hunk 1 above)
Comment (StanevPrime) [reply packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml]: Done

**Review by PrimeSwiftArt — APPROVED** [review]

**Pull request merged**